packages/kit/src/types/internal.d.ts

```diff
@@ -120,12 +126,18 @@ export class InternalServer extends Server {
 		options: RequestOptions & {
 			prerendering?: PrerenderOptions;
 			read: (file: string) => Buffer;
+			before_handle?: (event: RequestEvent, config: any, prerender: PrerenderOption) => void;
```

Comment:
this could use a comment explaining it's a hook called before handle

Reply:
```suggestion
			/** A hook called before `handle` during dev, so that `AsyncLocalStorage` can be populated */
			before_handle?: (event: RequestEvent, config: any, prerender: PrerenderOption) => void;
```